draft-ietf-httpbis-client-hints.md

```diff
@@ -50,6 +50,7 @@ normative:
       organization: Google
   W3C.REC-html5-20141028:
   W3C.CR-css-values-3-20150611:
+#  W3C.REC-CSS2-20110607:
```

Comment:
Yes, it seems the reference is borked. I believe the only way to fix this right now is to inline the full text...